README.md

```diff
@@ -33,14 +33,9 @@ $ npm install yarn -g
 $ yarn
 ```
 
-For easy production usage there is an additional package for managing NodeJS
-processes, `pm2`. This tool can run your application as a daemon, monitor
-occupied resources, gather logs and provide simple console interface for
-managing state of the app. To install it globally into your system run:
-
-```
-# npm install pm2 -g
-```
+The app is built into self-contained bundle for NodeJS server. There is
+a prepared SystemD unit file for control over the app. Another option is
+to use `pm2` process manager, but we had some problems with it.
```

Comment:
Would you mind specifying what problems? It might prove to be useful to someone.

Reply:
That piece of crap won't start after boot for no reason ... on one of three computers.

Reply:
Yes exactly :-) write it in readme file